### PR TITLE
Tweaks to README

### PR DESCRIPTION
**Pull request by wsargent**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,6 @@ This project contains the following top level components:
 * [OpenTelemetry API](api/):
   * [stable apis](api/all/src/main/java/io/opentelemetry/api/) including `Tracer`, `Span`, `SpanContext`, and `Baggage`
   * [semantic conventions](semconv/) Generated code for the OpenTelemetry semantic conventions.
-  * [context api](api/context/src/main/java/io/opentelemetry/context/) The OpenTelmetry Context implementation.
```
Comment (wsargent) [inline README.md:40]: context api directory does not exist

File: README.md [hunk 2]
```diff
@@ -67,7 +66,7 @@ Please refer to the [contribution guide](CONTRIBUTING.md) on how to set up for d
 
 ## Quick Start
 
-Please refer to the [quick start guide](QUICKSTART.md) on how use the OpenTelemetry API.
+Please refer to the [quick start guide](QUICKSTART.md) on how to use the OpenTelemetry API.
```
Comment (wsargent) [inline README.md:69]: missing word

File: README.md (hunk 1 above)
Comment (jkwatson) [reply README.md:40]: we shouldn't delete this line. Let's correct it to the correct directory:
`context/src/main/java/io/lopentelemetry/context/`

Comment (jkwatson) [reply README.md:40]: @wsargent bump!

Comment (wsargent) [reply README.md:40]: looking

Comment (wsargent) [reply README.md:40]: done

File: README.md [hunk 3]
```diff
@@ -37,8 +37,8 @@ This project contains the following top level components:
 * [OpenTelemetry API](api/):
   * [stable apis](api/all/src/main/java/io/opentelemetry/api/) including `Tracer`, `Span`, `SpanContext`, and `Baggage`
   * [semantic conventions](semconv/) Generated code for the OpenTelemetry semantic conventions.
-  * [context api](api/context/src/main/java/io/opentelemetry/context/) The OpenTelmetry Context implementation.
   * [metrics api](api/metrics/src/main/java/io/opentelemetry/api/metrics/) alpha code for the metrics API.
+* [context api](context/src/main/java/io/opentelemetry/context/) The OpenTelemetry Context implementation.
```
Comment (jkwatson) [inline README.md:41]: this is technically part of the "OpenTelemetry API" even though it's not in the `api` directory. I *think* it should be up in the above list, and included in the list of "stable apis" as well.

**Review by jkwatson — APPROVED** [review]
Thanks for the cleanup!

**Pull request merged**